examples/Consumer/Program.cs

```diff
@@ -2,45 +2,52 @@
 using Memphis.Client.Core;
 using System.Text.Json;
 
+MemphisClient? memphisClient = null;
+
 try
 {
     var options = MemphisClientFactory.GetDefaultOptions();
-    options.Host = "aws-us-east-1.cloud.memphis.dev";
-    options.AccountId = int.Parse(Environment.GetEnvironmentVariable("memphis_account_id"));
-    options.Username = "test_user";
-    options.Password = Environment.GetEnvironmentVariable("memphis_pass");
+    options.Host = "<memphis-host>";
+    // options.AccountId = "<memphis-accountId>";
```

Comment:
why is it in a comment? maybe needs another comment - for cloud users

Reply:
@shay23b because it errors if I don't put it in a comment. I don't want to put it in a string because it's not a string (it's an int), but I also can't leave it uncommented because it will error. 

I looked at the old examples and this was how this was treated to get past the linter complaining about the quotes